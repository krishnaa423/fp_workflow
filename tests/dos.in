&DOS
outdir='./tmp'
prefix='struct'
fildos='struct_dos.dat'
/
