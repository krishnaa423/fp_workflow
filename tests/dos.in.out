
     Program DOS v.7.2 starts on 17Jul2024 at  5:10:23 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors

     MPI processes distributed on     1 nodes
     28147 MiB available memory on the printing compute node when the environment starts
 

     Reading xml data from directory:

     ./tmp/struct.save/

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation= PBE
                           (   1   4   3   4   0   0   0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want

 
     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum         295     295     91                 3287     3287     609
 
     Using Pencil Decomposition
 

     Gaussian broadening (default values): ngauss,degauss=   0    0.000735

 
     DOS          :      0.07s CPU      0.13s WALL

 
   This run was terminated on:   5:10:23  17Jul2024            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
Warning: ieee_invalid is signaling
Warning: ieee_inexact is signaling
FORTRAN STOP
